liquid-staking/interactor/src/liquid_staking_interactor.rs

```diff
@@ -1,15 +1,16 @@
 #![allow(non_snake_case)]
 
-mod delegation_proxy;
 mod liquid_staking_config;
 mod liquid_staking_state;
 mod proxy;
 
+use delegation_sc_interact::DelegateCallsInteract;
 pub use liquid_staking_config::Config;
 use liquid_staking_state::State;
 use multiversx_sc_snippets::imports::*;
+use multiversx_sc_snippets::sdk::gateway::NetworkStatusRequest;
 
-const DELEGATION_MOCK_CONTRACT_CODE: &str = "../delegation-mock/output/delegation-mock.mxsc.json";
+pub const CHAIN_SIMULATOR_GATEWAY: &str = "http://localhost:8085";
```

Comment:
why?

Reply:
gateway fro local testing. I can also switch on devnet if needed.
Also we don't use mock contracts any more, so that part is not needed